resources/assets/coffee/react/_components/beatmapset-mapping.coffee

```diff
@@ -20,7 +20,7 @@ el = React.createElement
 
 bn = 'beatmapset-mapping'
 
-@BeatmapsetMapping = ({user, beatmapset}) ->
+@BeatmapsetMapping = ({user, beatmapset, beatmap}) ->
```

Comment:
apparently this should show the beatmapset state anyway (said the guy responsible for this).